src/InfluxDB.Collector/Util/EnumerableExtensions.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace InfluxDB.Collector.Util
+{
+    internal static class EnumerableExtensions
+    {
+        // from https://github.com/morelinq/MoreLINQ/blob/master/MoreLinq/Batch.cs
+        public static IEnumerable<IEnumerable<TSource>> Batch<TSource>(this IEnumerable<TSource> source, int size)
```

Comment:
To comply with the license of the original code, the original copyright info has to be included here. (Apache License 2.0 section 4.c.).